draft-packages/select/KaizenDraft/Select/README.mdx

```diff
@@ -82,6 +82,10 @@ Use a pre-selected good default where possible. In some cases, it cuts down on u
 *   **[Focus indicator](/guidelines/interaction-states#focus-indicator):**
       *   On keyboard navigation and click show focus indicator on the select field.
       *   For select options show focus indicator on keyboard navigation and on click (not hover).
+ *   **[Autocomplete](/components/autocomplete):**
+      *   Base select field should indicate that it is a search field, either through hint text, a search icon, or both.
```

Comment:
@lindsey-collins  in what I wrote up I've made a distinction between a searchable select (which sets a value) and a "search field" which returns results. Does that sound right or do you think we should treat searchable selects as a type of search field?

Reply:
@dlimiter I do agree with you now that I think about it! It could go either way but I think it's most clear to draw a line between these two items and treat them as different things. 

To clarify, in terms of this clash between what we've both written, is it my phrasing of "search field" (which I'm happy to change) or is it the entire concept of including hint text/the icon?

Reply:
@lindsey-collins Yep all I was concerned about was the term 'search field'. I think some kind of visual affordance that you can type to narrow the options is plus ultra 👍

Reply:
@dlimiter Awesome, thanks for clarifying! I _think_ I've changed that now - updated from 'search field' to 'searchable select'.

Reply:
👍